scripts/checkProgress1d.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cmd1="sqlite3 data/crypto_trading.db"
+sql0=".mode column"
+sql00=".headers on"
+sql1="SELECT th1.alt_coin_id AS coin, th1.alt_trade_amount AS amount, th1.crypto_trade_amount AS priceInUSD, (th1.alt_trade_amount - (SELECT th2.alt_trade_amount FROM trade_history th2 WHERE th2.alt_coin_id = th1.alt_coin_id AND th1.datetime > th2.datetime AND th2.selling = 0 ORDER BY th2.datetime DESC LIMIT 1)) AS change, datetime FROM trade_history th1 WHERE th1.state = 'COMPLETE' AND th1.selling = 0 AND th1.datetime > date('now', '-1 day') ORDER BY th1.datetime DESC;"
```

Comment:
Rather than outputting the last trades for the day, consider limiting the output size and only sending the last 10 trades e.g. (apprise has limits on message size)